neuralmonkey/processors/helpers.py

```diff
@@ -17,3 +17,14 @@ def untruecase(
             yield [sentence[0].capitalize()] + sentence[1:]
         else:
             yield []
+
+
+def pipeline(processors: List[Callable]) -> Callable:
```

Comment:
Takže v datasetu dáš jako preprocess <něco> a v [něco] dáš `class=pipeline` a jako `processors` tomu dáš seznam preprocess objektů?

Nebylo by lepší to nějak vyřešit přímo v tom datasetu? Jakože teď je to nějaká n-tice nebo co, tak přidat možnost do tý entice dát místo jednoho procesoru seznam?